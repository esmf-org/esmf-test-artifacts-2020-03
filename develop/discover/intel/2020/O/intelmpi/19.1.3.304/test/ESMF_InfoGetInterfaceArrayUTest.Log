build time -- 2021-10-05 01:18:50.546421
20211005 014127.229 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20211005 014127.230 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20211005 014127.230 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20211005 014127.230 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20211005 014127.230 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20211005 014127.230 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20211005 014127.230 INFO             PET0 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_20-18-g0aeed17f50
20211005 014127.230 INFO             PET0 ESMF library build date/time: "Oct  5 2021" "01:15:59"
20211005 014127.230 INFO             PET0 ESMF library build location : /gpfsm/dnb04/projects/p98/mpotts/esmf/intel_2020_intelmpi_O_develop
20211005 014127.230 INFO             PET0 ESMF_COMM                   : intelmpi
20211005 014127.231 INFO             PET0 ESMF_MOAB                   : enabled
20211005 014127.231 INFO             PET0 ESMF_LAPACK                 : enabled
20211005 014127.231 INFO             PET0 ESMF_NETCDF                 : enabled
20211005 014127.231 INFO             PET0 ESMF_PNETCDF                : disabled
20211005 014127.231 INFO             PET0 ESMF_PIO                    : enabled
20211005 014127.231 INFO             PET0 ESMF_YAMLCPP                : enabled
20211005 014127.232 INFO             PET0  Beginning Test, file ESMF_InfoGetInterfaceArrayUTest.F90, line 61
20211005 014127.232 INFO             PET0  NUMBER_OF_PROCESSORS           4
20211005 014127.242 INFO             PET0  PASS  Get and manipulate an Array attribute, ESMF_InfoGetInterfaceArrayUTest.F90, line 117
20211005 014127.242 INFO             PET0  PASS  Deleting a host object and using a handle/view, ESMF_InfoGetInterfaceArrayUTest.F90, line 142
20211005 014127.242 INFO             PET0  Ending Test, file ESMF_InfoGetInterfaceArrayUTest.F90, line 149
20211005 014127.242 INFO             PET0 Finalizing ESMF
20211005 014127.229 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20211005 014127.230 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20211005 014127.230 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20211005 014127.230 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20211005 014127.230 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20211005 014127.230 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20211005 014127.230 INFO             PET1 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_20-18-g0aeed17f50
20211005 014127.230 INFO             PET1 ESMF library build date/time: "Oct  5 2021" "01:15:59"
20211005 014127.230 INFO             PET1 ESMF library build location : /gpfsm/dnb04/projects/p98/mpotts/esmf/intel_2020_intelmpi_O_develop
20211005 014127.230 INFO             PET1 ESMF_COMM                   : intelmpi
20211005 014127.231 INFO             PET1 ESMF_MOAB                   : enabled
20211005 014127.231 INFO             PET1 ESMF_LAPACK                 : enabled
20211005 014127.231 INFO             PET1 ESMF_NETCDF                 : enabled
20211005 014127.231 INFO             PET1 ESMF_PNETCDF                : disabled
20211005 014127.231 INFO             PET1 ESMF_PIO                    : enabled
20211005 014127.231 INFO             PET1 ESMF_YAMLCPP                : enabled
20211005 014127.232 INFO             PET1  Beginning Test, file ESMF_InfoGetInterfaceArrayUTest.F90, line 61
20211005 014127.232 INFO             PET1  NUMBER_OF_PROCESSORS           4
20211005 014127.242 INFO             PET1  PASS  Get and manipulate an Array attribute, ESMF_InfoGetInterfaceArrayUTest.F90, line 117
20211005 014127.242 INFO             PET1  PASS  Deleting a host object and using a handle/view, ESMF_InfoGetInterfaceArrayUTest.F90, line 142
20211005 014127.242 INFO             PET1  Ending Test, file ESMF_InfoGetInterfaceArrayUTest.F90, line 149
20211005 014127.242 INFO             PET1 Finalizing ESMF
20211005 014127.230 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20211005 014127.230 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20211005 014127.230 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20211005 014127.230 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20211005 014127.230 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20211005 014127.230 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20211005 014127.230 INFO             PET2 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_20-18-g0aeed17f50
20211005 014127.230 INFO             PET2 ESMF library build date/time: "Oct  5 2021" "01:15:59"
20211005 014127.230 INFO             PET2 ESMF library build location : /gpfsm/dnb04/projects/p98/mpotts/esmf/intel_2020_intelmpi_O_develop
20211005 014127.230 INFO             PET2 ESMF_COMM                   : intelmpi
20211005 014127.232 INFO             PET2 ESMF_MOAB                   : enabled
20211005 014127.232 INFO             PET2 ESMF_LAPACK                 : enabled
20211005 014127.232 INFO             PET2 ESMF_NETCDF                 : enabled
20211005 014127.232 INFO             PET2 ESMF_PNETCDF                : disabled
20211005 014127.232 INFO             PET2 ESMF_PIO                    : enabled
20211005 014127.232 INFO             PET2 ESMF_YAMLCPP                : enabled
20211005 014127.232 INFO             PET2  Beginning Test, file ESMF_InfoGetInterfaceArrayUTest.F90, line 61
20211005 014127.232 INFO             PET2  NUMBER_OF_PROCESSORS           4
20211005 014127.243 INFO             PET2  PASS  Get and manipulate an Array attribute, ESMF_InfoGetInterfaceArrayUTest.F90, line 117
20211005 014127.243 INFO             PET2  PASS  Deleting a host object and using a handle/view, ESMF_InfoGetInterfaceArrayUTest.F90, line 142
20211005 014127.243 INFO             PET2  Ending Test, file ESMF_InfoGetInterfaceArrayUTest.F90, line 149
20211005 014127.243 INFO             PET2 Finalizing ESMF
20211005 014127.231 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20211005 014127.231 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20211005 014127.231 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20211005 014127.231 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20211005 014127.231 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20211005 014127.231 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20211005 014127.231 INFO             PET3 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_20-18-g0aeed17f50
20211005 014127.231 INFO             PET3 ESMF library build date/time: "Oct  5 2021" "01:15:59"
20211005 014127.231 INFO             PET3 ESMF library build location : /gpfsm/dnb04/projects/p98/mpotts/esmf/intel_2020_intelmpi_O_develop
20211005 014127.231 INFO             PET3 ESMF_COMM                   : intelmpi
20211005 014127.232 INFO             PET3 ESMF_MOAB                   : enabled
20211005 014127.232 INFO             PET3 ESMF_LAPACK                 : enabled
20211005 014127.232 INFO             PET3 ESMF_NETCDF                 : enabled
20211005 014127.232 INFO             PET3 ESMF_PNETCDF                : disabled
20211005 014127.232 INFO             PET3 ESMF_PIO                    : enabled
20211005 014127.232 INFO             PET3 ESMF_YAMLCPP                : enabled
20211005 014127.232 INFO             PET3  Beginning Test, file ESMF_InfoGetInterfaceArrayUTest.F90, line 61
20211005 014127.232 INFO             PET3  NUMBER_OF_PROCESSORS           4
20211005 014127.241 INFO             PET3  PASS  Get and manipulate an Array attribute, ESMF_InfoGetInterfaceArrayUTest.F90, line 117
20211005 014127.242 INFO             PET3  PASS  Deleting a host object and using a handle/view, ESMF_InfoGetInterfaceArrayUTest.F90, line 142
20211005 014127.242 INFO             PET3  Ending Test, file ESMF_InfoGetInterfaceArrayUTest.F90, line 149
20211005 014127.242 INFO             PET3 Finalizing ESMF
