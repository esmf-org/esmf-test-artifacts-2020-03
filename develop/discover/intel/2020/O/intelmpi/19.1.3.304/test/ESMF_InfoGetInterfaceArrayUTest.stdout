build time -- 2021-11-01 01:09:30.406455
  Beginning Test, file ESMF_InfoGetInterfaceArrayUTest.F90, line 61
  Beginning Test, file ESMF_InfoGetInterfaceArrayUTest.F90, line 61
  Beginning Test, file ESMF_InfoGetInterfaceArrayUTest.F90, line 61
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_InfoGetInterfaceArrayUTest.F90, line 61
  NUMBER_OF_PROCESSORS           4
  NUMBER_OF_PROCESSORS           4
  NUMBER_OF_PROCESSORS           4
  PASS  Get and manipulate an Array attribute, ESMF_InfoGetInterfaceArrayUTest.F
 90, line 117
  PASS  Deleting a host object and using a handle/view, ESMF_InfoGetInterfaceArr
 ayUTest.F90, line 142
  Ending Test, file ESMF_InfoGetInterfaceArrayUTest.F90, line 149
  PASS  Get and manipulate an Array attribute, ESMF_InfoGetInterfaceArrayUTest.F
 90, line 117
  PASS  Get and manipulate an Array attribute, ESMF_InfoGetInterfaceArrayUTest.F
 90, line 117
  PASS  Deleting a host object and using a handle/view, ESMF_InfoGetInterfaceArr
 ayUTest.F90, line 142
  Ending Test, file ESMF_InfoGetInterfaceArrayUTest.F90, line 149
  PASS  Deleting a host object and using a handle/view, ESMF_InfoGetInterfaceArr
 ayUTest.F90, line 142
  Ending Test, file ESMF_InfoGetInterfaceArrayUTest.F90, line 149
  PASS  Get and manipulate an Array attribute, ESMF_InfoGetInterfaceArrayUTest.F
 90, line 117
  PASS  Deleting a host object and using a handle/view, ESMF_InfoGetInterfaceArr
 ayUTest.F90, line 142
  Ending Test, file ESMF_InfoGetInterfaceArrayUTest.F90, line 149
  PET           1  Test Elapsed Time    16.3760000000000       msec.
  PET           3  Test Elapsed Time    24.9240000000000       msec.
  PET           0  Test Elapsed Time    21.6500000000000       msec.
  PET           2  Test Elapsed Time    21.4800000000000       msec.
